extension/bootstrap.js

```diff
@@ -242,25 +241,6 @@ function uninstall(data, reason) {
         .getService(Components.interfaces.nsIPrefService).getBranch("extensions.personas.").deleteBranch("");
     Components.classes["@mozilla.org/intl/stringbundle;1"]
         .getService(Components.interfaces.nsIStringBundleService).flushBundles();
```

Comment:
I feel like those lines could be included in `shutdown` above as well and we could eliminate this function. Out of the scope for this bug though, but a thing we can keep in mind.